src/node/node_state.h

```diff
@@ -976,7 +976,7 @@ namespace ccf
           {
             GetQuotes::Quote q;
             q.node_id = nid;
-            q.raw = ni.quote;
+            q.raw = fmt::format("{:02x}", fmt::join(ni.quote, ""));
```

Comment:
Curious: what motivated hex over base 64 here?

Reply:
Consistency with the nearby mrenclave which is already hex, and doing the minimal useful thing. base64 may be better (it has an official OpenAPI format, which hex doesn't), but requires a fatter parser.